integration/lifecycle/lifecycle_test.go

```diff
@@ -66,8 +67,20 @@ var _ = Describe("Lifecycle", func() {
 		core.Ledger.State.StateDatabase = "CouchDB"
 		core.Ledger.State.CouchDBConfig.CouchDBAddress = couchAddr
 		processes[couchDB.Name] = couchProcess
+		org1TermFile := filepath.Join(testDir, "org1-term-file")
+		os.Setenv("ORG1_TERM_FILE", org1TermFile)
+		core.Chaincode.ExternalBuilders[0].EnvironmentWhitelist = append(core.Chaincode.ExternalBuilders[0].EnvironmentWhitelist, "ORG1_TERM_FILE")
```

Comment:
Is there always one external builder? (Honestly don't remember.) If not, we probably need more than referencing index 0.

Reply:
nwo.New creates a network with just one external builder.

Reply:
Sure, and while true today, it may not be generally true. I commented on this because it's fragile. We don't have to hit it here but we should avoid this kind of thing in the future.

If you're using a topology or config that's generated from a shared source, it can change.

Reply:
Done